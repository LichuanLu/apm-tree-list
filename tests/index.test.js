// import pkg from '../package.json';

describe('component dist files', () => {
  // https://github.com/ant-design/ant-design/issues/1638
  // https://github.com/ant-design/ant-design/issues/1968
  it('exports modules correctly', () => {
    const components = require('../src/components'); // eslint-disable-line global-require
    expect(Object.keys(components)).toMatchSnapshot();
  });

  // https://github.com/ant-design/ant-design/issues/1970
  // https://github.com/ant-design/ant-design/issues/1804
  // if (process.env.CI) {
  //   it('should have antd.version', () => {
  //     const antd = require('../dist/antd'); // eslint-disable-line global-require
  //     expect(antd.version).toBe(pkg.version);
  //   });
  // }
});
