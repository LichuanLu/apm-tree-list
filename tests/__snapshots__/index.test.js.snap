// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component dist files exports modules correctly 1`] = `
Array [
  "TreeList",
  "TreeListUtils",
  "TreeSelector",
  "TreeSelectorItem",
]
`;
